frontend/src/components/ScheduledEventsRunHistoryTableCard/utils/index.ts

```diff
@@ -0,0 +1,24 @@
+import * as localForage from 'localforage';
+import { api, localForageKeys } from '../../../utils';
+import axios from 'axios';
+
+const BASEPATH = api.routes.VIEW_SCHEDULED_EVENTS;
```

Comment:
I don't think its a problem how you guys are using this ... these endpoints could change, and then you'd find yourself with lots of broken features ... So for each endpoint, I'm gonna need a separate constant, not one concatenated from another.
i.e. have one for `FETCH_RUN_INSTANCES` ... if you're doing this somewhere else, please fix ... I saw this in David's code as well & think it originated here.